avaje-config/src/main/java/io/avaje/config/InitialLoader.java

```diff
@@ -204,6 +204,19 @@ private void loadViaIndirection() {
     }
   }
 
+  /** Load configuration defined by a <em>config.profiles</em> property. */
+  private void loadViaProfiles() {
+    final String paths = loadContext.profiles();
+    if (paths != null) {
+      for (final String path : splitPaths(paths)) {
+        final var profile = loadContext.eval(path);
+        loadProperties("application-" + profile + ".properties", RESOURCE);
+        loadYaml("application-" + profile + ".yaml", RESOURCE);
+        loadYaml("application-" + profile + ".yml", RESOURCE);
+      }
```

Comment:
Just wondering if this should load FILE as well as RESOURCE?

I don't think there is any harm in trying to load FILE for these as well. Hmm.

Reply:
ok